### ISSUE TITLE
aria-valuenow has non-integer values

### ISSUE THREAD

**Issue by gsilver** [kaltura/mwEmbed#4315]
In scrubber.js there is the following

'aria-valuenow': parseInt(perc * 100) + '%',

which resolves to aria-valuenow="N%" 

aria-valuenow accepts only numeric values so that the current position can be reported to assistive tech. As it stands assistive tech does not have access to this data

Thanks!

**Comment by robfentress** [comment]
This is important to us as well.

**Comment by robfentress** [comment]
I believe [aria-valuenow](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-valuenow) should be a number that is between `aria-valuemin` and `aria-valuemax`.  That means that if the video was five minutes long and you were two minutes into it, the value of `aria-valuenow` would be `120`.  Assuming this pattern was well supported by assistive technologies, you could follow the [Media Seek Slider Example](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-seek/) from the ARIA Authoring Practices Guide.

**Comment by tadsanwork** [comment]
FWIW, W3C says decimal to account for non-whole value.  Fact still remains that it shouldn't be a % - https://www.w3.org/TR/wai-aria-1.2/#aria-valuenow

![Image](https://github.com/user-attachments/assets/80a4a07a-4b66-4278-9ca5-782cfb502352)
